shared/lib_pvshade.cpp

```diff
@@ -342,8 +365,9 @@ bool ss_exec(
 	double albedo,		// used to calculate reduced relected irradiance
 	bool trackmode,		// 0 for fixed tilt, 1 for one-axis tracking
 	bool linear,		// 0 for non-linear shading (C. Deline's full algorithm), 1 to stop at linear shading
-	double shade_frac_1x,	// geometric calculation of the fraction of one-axis row that is shaded (0-1), not used if fixed tilt 
+	double shade_frac_1x,	// geometric calculation of the fraction of one-axis row that is shaded (0-1), not used if fixed tilt
 
+	ssSkyDiff &skydiffs,
```

Comment:
please include detailed comment of what this is, so folks using the function know what they're expected to pass in and how to initialize it